### PR TITLE
[CSM-249] Wallet renamings

### PR DESCRIPTION
**Pull request by Martoon-00**
This PR moves wallet code to a new convention, from `walletSet -> wallet -> account` to `wallet -> account -> address` model.

Scripts which performed renaming are preserved in CSV, so one can replay / modify performed changes quite easily

### CODE REVIEW

**Comment by Martoon-00** [comment]
Currently renamed: types, fields, methods.
What remains: local variables

@akegalj please take a short look at it. If you like new names, then I proceed to next step, while one who works on frontend can make required changes.

I'll also check I didn't miss anything at noon

**Comment by akegalj** [comment]
@ksaric @denisshevchenko before this is merged (or after) we should update the docs https://cardanodocs.com/technical/wallet-frontend/ with the new terminology - so we should either:
 * rerun the commands to produce the output
 * use rename script made by @Martoon-00 to rename that file

@Martoon-00 In order not to train other people to use the script - would you mind running rename script that you run to this PR also to that mardown file in cardanodocs ?

**Comment by Martoon-00** [comment]
Ohoh, it's in another repository, right?

**Comment by akegalj** [comment]
@Martoon-00 right:
```
[akegalj@notebook cardanodocs.com]$ git remote -v
origin	git@github.com:input-output-hk/cardanodocs.com.git (fetch)
origin	git@github.com:input-output-hk/cardanodocs.com.git (push)

```

**Comment by Martoon-00** [comment]
I forgot to rename methods names in daedalus, now should be ok

**Comment by Martoon-00** [comment]
Noooooo

**Comment by Martoon-00** [comment]
@akegalj please :white_check_mark:

File: daedalus/src/Daedalus/BackendApi.purs [hunk 1]
```diff
@@ -107,52 +107,52 @@ testReset :: forall eff. Aff (ajax :: AJAX | eff) Unit
...
-deleteWallet wId = deleteR $ noQueryParam ["wallets", walletAddressToUrl wId]
+deleteAccount :: forall eff. CAccountId -> Aff (ajax :: AJAX | eff) Unit
+deleteAccount wId = deleteR $ noQueryParam ["wallets", walletAddressToUrl wId]
 
 --------------------------------------------------------------------------------
 -- Accounts --------------------------------------------------------------------
 
-newAccount :: forall eff. Maybe CPassPhrase -> CWalletAddress -> Aff (ajax :: AJAX | eff) CAccount
-newAccount pass = postRBody $ queryParams ["account"] [qParam "passphrase" $ _passPhrase <$> pass]
+newWAddress :: forall eff. Maybe CPassPhrase -> CAccountId -> Aff (ajax :: AJAX | eff) CAddress
```
Comment (akegalj) [inline daedalus/src/Daedalus/BackendApi.purs]: isn't `newAccount` better naming? (as it returns CAccount)

Comment (Martoon-00) [inline daedalus/src/Daedalus/BackendApi.purs]: It returns `CAddress`

Comment (akegalj) [inline daedalus/src/Daedalus/BackendApi.purs]: aha, ok then

**Pull request merged**